index.html

```diff
@@ -8,11 +8,31 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link rel="stylesheet" href="css/fonts.css">
 </head>
 <body>
   <header>
+    <div class="container">
+      <nav class="navbar">
+        <div class="flex-group navbar-left">
+          <a class="logo">GameQueue</a>
```

Comment:
Needs an `href` attribute. Can just be set to a `#` to prevent the page from jumping somewhere.